tests/stability/test_ols.py

```diff
@@ -0,0 +1,12 @@
+import dask.array as da
+
+from ..utils_test import wait
+
+
+def test_ols(small_client, num_samples=int(1e8), num_coeffs=100, chunksize=int(1e6)):
```

Comment:
Nit: While defining `num_samples`, `num_coeffs`, and `chunksize` as keyword arguments seems to be working, this isn't typically done when writing `pytest`-style tests. Instead I'd recommend something like (note, I've not run linters on the code below):

```suggestion
def test_ols(small_client):
    num_samples = int(1e8)
    num_coeffs = 100
    chunksize = int(1e6)
```

which is more `pytest`-idiomatic. This should reduce the amount of thought future readers of this test will need.